frontend/src/components/requisitionForm/RequisitionItemCard.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import { Button, Card, Col, Form, Input, InputNumber, Row, Tooltip } from "antd";
+import { QuestionCircleOutlined } from "@ant-design/icons/lib";
+
+const { TextArea } = Input;
+
+interface FieldData {
+  name: number;
+  key: number;
+  fieldKey: number;
+}
+
+interface Props {
+  field: FieldData;
+  deleteButton: boolean;
+  rules: any;
+  remove: (index: number) => void;
+}
+
+const RequisitionItemCard: React.FC<Props> = (props) => (
+  <Card
+    size="small"
+    title={`Item ${props.field.name + 1}`}
+    style={{ marginBottom: "15px" }}
+    extra={props.deleteButton
+      ? (
+        <Button onClick={() => props.remove(props.field.name)} type="text" size="small" danger>
+          Delete
+        </Button>
+      ) : null}
```

Comment:
You should be able shorten this to
```suggestion
    extra={props.deleteButton && (
        <Button onClick={() => props.remove(props.field.name)} type="text" size="small" danger>
          Delete
        </Button>
      )}
```